core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -103,11 +103,14 @@ private[log] class LogCleanerManager(val logDirs: Seq[File],
             val lastClean = allCleanerCheckpoints
             val now = Time.SYSTEM.milliseconds
             partitions.iterator.map { tp =>
-              val log = logs.get(tp)
-              val lastCleanOffset = lastClean.get(tp)
-              val offsetsToClean = cleanableOffsets(log, lastCleanOffset, now)
-              val (_, uncleanableBytes) = calculateCleanableBytes(log, offsetsToClean.firstDirtyOffset, offsetsToClean.firstUncleanableDirtyOffset)
-              uncleanableBytes
+              Option(logs.get(tp)).map(
```

Comment:
We could do the following the make the code a bit simpler and more consistent.

```
 map { log =>
                  val lastCleanOffset = lastClean.get(tp)
                  ...
          }

```

Reply:
Fixed.